includes/admin/class-wp-job-manager-settings.php

```diff
@@ -528,6 +528,45 @@ protected function init_settings() {
 						],
 					],
 				],
+				'stats'          => [
+					__( 'Stats', 'wp-job-manager' ),
+					[
+						[
+							'name'       => 'job_manager_stats_enable',
+							'std'        => '0',
+							'label'      => __( 'Enable stats', 'wp-job-manager' ),
+							'cb_label'   => __( 'Enable stats', 'wp-job-manager' ),
+							'desc'       => __( 'Enable recording various stats (e.g. listing views) and showing them to employers', 'wp-job-manager' ),
+							'type'       => 'checkbox',
+							'attributes' => [],
+						],
+						[
+							'name'       => 'job_manager_stats_require_paid_listing',
+							'std'        => '0',
+							'label'      => __( 'Require paid listing', 'wp-job-manager' ),
+							'cb_label'   => __( 'Require paid listing', 'wp-job-manager' ),
+							'desc'       => __( 'Only display stats to employers that purchased a paid-listing package', 'wp-job-manager' ),
+							'type'       => 'checkbox',
+							'attributes' => [],
+						],
+						[
+							'name'       => 'job_manager_stats_default_date_range',
+							'std'        => '30',
+							'label'      => __( 'Stats default date range', 'wp-job-manager' ),
+							'desc'       => __( 'The default date range (in days) of stats to display in the frontend dashboard', 'wp-job-manager' ),
+							'type'       => 'number',
+							'attributes' => [],
```

Comment:
Can this be customized by the user using the frontend dashboard? Like, can it have a maximum?

Reply:
yeah I think it will, agree probably needs to have a max.